idaes/core/base/flowsheet_model.py

```diff
@@ -275,9 +275,10 @@ def visualize(self, model_name, **kwargs):
             **kwargs: Additional keywords for :func:`idaes.core.ui.fv.visualize()`
 
         Returns:
-            None
+            VisualizeResult : The return from UI, namedtuple("VisualizeResult", ["store", "port", "server", "save_diagram"]) etc...
```

Comment:
Using the `class` and `meth` Sphinx roles for the Python domain (https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-class) will generate clickable references in the Sphinx docs for this module (the references might not be enabled until the Intersphinx machinery is done, but it should not generate an error before that).

```suggestion
            visualize_result: The :class:`idaes_ui.fv.fsvis.VisualizeResult` instance returned by :meth:`UI.visualize`
```